playground/infrastructure/cd_helper.py

```diff
@@ -26,6 +26,7 @@
 from pathlib import Path
 from typing import List
 
+from tqdm import tqdm
```

Comment:
do we need to add this `tqdm` as a dependency somewhere?

Reply:
I added this dependency to the requirements list `requirements.txt`.